### PR TITLE
Update bash quiz

### PR DESCRIPTION
**Pull request by DevHabit**
## What's new?

Fixed format issue.
Replaced images of Q30 with higher quality images.

### CODE REVIEW

File: bash/bash-quiz.md [hunk 1]
```diff
@@ -35,10 +35,10 @@ done
...
 #### Q3. To run a copy command in a subshell, which syntax would you use?
 
-- [x] ( command )
-- [ ] sh command
-- [ ] { command; }
-- [ ] (( command ))
+- [x] `**( command )**`
+- [ ] `**sh command**`
+- [ ] `**{ command; }**`
+- [ ] `**(( command ))**`
```
Comment (Ebazhanov) [inline bash/bash-quiz.md:41]:
```suggestion
- [x] `( command )`
- [ ] `sh command`
- [ ] `{ command; }`
- [ ] `(( command ))`
```
I guess we do not need additional asterics, what do you think @DevHabit ?

**Review by Ebazhanov — CHANGES_REQUESTED** [review]

File: bash/bash-quiz.md (hunk 1 above)
Comment (DevHabit) [reply bash/bash-quiz.md]: Hi, @Ebazhanov .
Addressed comments.

Comment (Ebazhanov) [reply bash/bash-quiz.md]: thanks for the quick reply! good

**Review by Ebazhanov — APPROVED** [review]
Thank you again @DevHabit, looks like you are a workaholic! :+1:

**Pull request merged**